beetsplug/bandcamp.py

```diff
@@ -141,6 +150,20 @@ def get_album_info(self, url):
             release = html.find('meta', attrs={'itemprop': 'datePublished'})['content']
             release = isodate.parse_date(release)
             artist_url = url.split('/album/')[0]
+
+            if self.config['description']:
+                try:
+                    import html2text
+                except ImportError:
+                    self._log.warning('Failed to import html2text')
+                    pass
+                else:
+                    description = html.find('div', attrs={'itemprop': 'description'})
+                    if description:
+                        totext = html2text.HTML2Text()
+                        description = totext.handle(str(description).strip())
+                        self.descriptions[album_id] = description
```

Comment:
Conceptually I'm not a fan of adding a side effect to this method, which normally just returns AlbumInfo objects. I may tweak this to add it to the AlbumInfo objects and save it in the class in the caller instead.

Reply:
It'd be nice if AlbumInfo and TrackInfo provided a built in mechanism to add extra fields to be applied if they end up used, then we could avoid the import stage for both lyrics and description. It may be worth proposing that in bandcamp upstream, to see if there's interest from any other plugin developers.